parts/ch11.tex

```diff
@@ -32,16 +34,16 @@ \subsection{Интенсиональность и экстенсионально
 
-Представьте себе, что подруга, взявшая у вас на время машину, звонит и говорит, что произошла авария машину занесло на мокрой дороге и она перевернулась, упав в кювет «Я чудом избежала смерти,» --- говорит она. В голове у вас появляются, одна за другой, соответствующие образы, которые становятся все реальнее по мере того как собеседница добавляет всё новые детали; в конце рассказа вся картина стоит у вас перед глазами. Вдруг она, смеясь, сообщает вам что все это --- первоапрельская шутка, и что ни с ней, ни с машиной ничего не случилось! В некотором смысле, это ничего не меняет. История и образы вызванные ею не теряют своей жизненности и надолго остаются у вас в памяти. В дальнейшей вы можете считать вашу подругу плохим водителем, поскольку впечатление оставленное её рассказом, не пропало, когда вы узнали, что это --- неправда. Выдумка и факт тесно переплетаются в нашем сознании, и это происходит потому, что мышление предполагает способность к изобретению сложных описаний и манипуляции ими, эти описания совсем не обязательно должны быть привязаны к реальным фактам или вещам.
+Представьте себе, что подруга, взявшая у вас на время машину, звонит и говорит, что произошла авария машину занесло на мокрой дороге и она перевернулась, упав в кювет... «Я чудом избежала смерти,» --- говорит она. В голове у вас появляются, одна за другой, соответствующие образы, которые становятся все реальнее по мере того как собеседница добавляет всё новые детали; в конце рассказа вся картина стоит у вас перед глазами. Вдруг она, смеясь, сообщает вам что все это --- первоапрельская шутка, и что ни с ней, ни с машиной ничего не случилось! В некотором смысле, это ничего не меняет. История и образы вызванные ею не теряют своей жизненности и надолго остаются у вас в памяти. В дальнейшей вы можете считать вашу подругу плохим водителем, поскольку впечатление оставленное её рассказом, не пропало, когда вы узнали, что это --- неправда. Выдумка и факт тесно переплетаются в нашем сознании, и это происходит потому, что мышление предполагает способность к изобретению сложных описаний и манипуляции ими, эти описания совсем не обязательно должны быть привязаны к реальным фактам или вещам.
```

Comment:
Replace "..." with "\ldots".
Also, replace ",»" with "»," (yeah, Russian language have some weird rules! see here 199: http://new.gramota.ru/biblio/readingroom/rules/165-oszp (random source)). Actually, it was already done in 2c76c487e8e4f0f800afff58b0c870da638e3120.

Reply:
Will be done.